certified-connectors/PDF4me Connect/apiDefinition.swagger.json

```diff
@@ -1482,8 +1903,8 @@
         "consumes": [
           "application/json"
         ],
-        "description": "Add a background watermark text to the PDF document. https://pdf4me.com/power-automate/documentation/actions/text-stamp/",
-        "operationId": "Stamp_V1",
```

Comment:
@varunynoox The operation has been deleted. This change breaks existing flows.
Can you please confirm that no users will be affected by this change?

Reply:
operation id has been restored with commit - https://github.com/microsoft/PowerPlatformConnectors/commit/685ab73dd86143b823f5c9b44bda06904cbe0508